packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -115,6 +116,20 @@ export interface VpcNetworkProps {
    * private subnet per AZ
    */
   subnetConfiguration?: SubnetConfiguration[];
+
+  /**
+   * Indicates whether a VPN gateway should be created and attached to this VPC.
+   *
+   * @default false
```

Comment:
Can we make this default smarter, so if I define vpn connections on this VPC, it will default to "true", and otherwise to "false". I would expect this to Just Work (TM):

```ts
new VpcNetwork(this, 'VPC', {
  vpnConnections: {
    dynamic: { ip: '1.2.3.4' }
  }
});
```